dev/io.openliberty.data.internal.persistence/resources/io/openliberty/data/internal/persistence/resources/CWWKDMessages.nlsprops

```diff
@@ -600,7 +600,7 @@ CWWKD1060.name.out.of.scope.useraction=Update the Repository annotation to \
  specify a data store name that is visible to the module or library \
  providing the repository interface.
 
-CWWKD1061.comp.name.in.ejb=CWWKD1061E: The {0} repository interface which is \
+CWWKD1061.comp.name.in.ejb=CWWKD1061E: The {0} repository interfaces which are \
  defined in the {1} enterprise bean module of the {2} application configures \
```

Comment:
This will also impact several other recently added messages.  Let's not change them in this PR because it isn't clear how best to do so and will further complicate things by requiring an ID review on top of adding merge conflicts.  Note that the change above, if made, would also require switching `configures` to `configure` due to the plural subject.  But for now, you can just skip the message updates by supplying the first interface name as the parameter.